Podfile.lock

```diff
@@ -2,17 +2,18 @@ PODS:
   - PromisesObjC (1.2.2)
   - PromisesSwift (1.2.2):
     - PromisesObjC (= 1.2.2)
-  - Reusable (4.0.2):
-    - Reusable/Storyboard (= 4.0.2)
-    - Reusable/View (= 4.0.2)
-  - Reusable/Storyboard (4.0.2)
-  - Reusable/View (4.0.2)
-  - Swinject (2.4.1)
-  - SwinjectStoryboard (2.0.1):
-    - Swinject (~> 2.4)
-  - TagListView (1.3.1)
+  - Reusable (4.0.0):
```

Comment:
This file should be entirely reverted (just take file content from the master and copy/paste it here)